### ISSUE TITLE
EIN fails to install from MELPA Stable

### ISSUE THREAD

**Issue by marcecj** [millejoh/emacs-ipython-notebook#597]
As the summary says, I cannot install EIN from MELPA Stable due several non-existent dependencies:

    Requires: emacs-25, websocket-20190620.338 (not available), auto-complete-1.4.0, request-20190621.1622 (not available), deferred-0.5, polymode-20190426.1729 (not available), markdown-mode-20171116.756 (not available), dash-2.13.0, s-1.11.0, skewer-mode-1.6.2

(They only exist in those versions in regular MELPA).

I hope this is the right place to file this issue.  Right now I've worked around it by adding MELPA to `package-sources` with a lower priority than MELPA Stable, but I think it would be better if installation worked out of the box, or for it to be made clear that EIN should only be installed from regular MELPA and *not* from MELPA Stable.

**Comment by millejoh** [comment]
You have the right place - and sorry about that. I will push an update and "release" a new version. I should also update travis to test the so-called stable version of ein.

**Comment by marcecj** [comment]
Sorry for the late reply.  I can install EIN from MELPA stable now, so from my perspective, this is now fixed.  Do you think this issue can now be closed?

**Comment by marcecj** [comment]
(Again with the tardiness...) Thanks!

**Comment by mikegilchrist** [comment]
I'm having this same problem. Trying to install from MELPA stable, but get
`package-compute-transaction: Need package ‘websocket-20191017.30’, but only 1.12 is available`

**Comment by dickmao** [comment]
If I knew how to remove EIN from melpa-stable, I would.  Only MELPA-unstable is supported.

**Comment by mikegilchrist** [comment]
Thanks for the quick and helpful response.  I will try installing the unstable version.
